### DISCUSSION TITLE
is the api at: /api/v1/ in spoke editable?

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#2875] [General Questions & Discussion] [answered]
Might be a noob question but just wondering since sometimes it gives me code 422 errors when trying to create a project in a custom spoke instance. I cant find the file which might control it in the source code.

**Comment by robinkwilson** [comment, marked as answer]
The /api/v1 endpoints are defined in reticulum: https://github.com/mozilla/reticulum/blob/master/lib/ret_web/router.ex

Right now, we don't have a way to update reticulum for your HC instance.
If your custom spoke instance is running locally, you're likely running against our dev servers where you don't have full access permissions.
